lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -53,7 +53,7 @@ public void TestTaskCloseOnLocalRuntime()
         {
             const int chunkSize = 2;
             const int dims = 50;
-            const int iterations = 200;
+            const int iterations = 1000;
```

Comment:
Should we modify test verification to stress that we expect all tasks to be closed rather than fail? We could increase the number of iterations even more in this case